_maps/map_catalogue.txt

```diff
@@ -462,6 +462,9 @@ Find the key for using this catalogue in "map_catalogue_key.txt"
 		Size = (x = 19)(y = 15)(z = 1)
 		Tags = "No Combat", "Minor Loot", "Shelter"
 
+		File Name = "_maps\RandomRuins\SpaceRuins\singularitylab.dmm"
+		Size = (x = 121)(y = 75)(z = 1)
+		Tags = "Boss Combat Challenge", "Major Loot", "Bad Shelter", "Shelter"
```

Comment:
Remove "Bad Shelter". Bad shelter would be like if you have a building with no air on a jungle planet. Reverse of shelter.